### PR TITLE
Add support for Swift 2.2

### PR DESCRIPTION
**Pull request by anthonycastelli**
Add support for Swift 2.2

### CODE REVIEW

File: Sources/Kitchen.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2015 toshi0383. All rights reserved.
 //
 
-@exported import TVMLKit
+import TVMLKit
```
Comment (toshi0383) [inline Sources/Kitchen.swift]: Should be `@_exported` ?

**Comment by toshi0383** [comment]
Wow you're so quick 😄

File: Sources/Kitchen.swift (hunk 1 above)
Comment (anthonycastelli) [inline Sources/Kitchen.swift]: Oh whoops! My bad.

**Comment by anthonycastelli** [comment]
haha been going through all my projects and upgrading to v2.2

**Comment by toshi0383** [comment]
All UnitTests has been passed on my Mac! 🚀

**Comment by toshi0383** [comment]
Going to create `swift2.2` branch and make it as default.

**Comment by anthonycastelli** [comment]
Sounds good :+1:

**Pull request merged**